tests/integration/update.bats

```diff
@@ -76,6 +78,10 @@ function check_cgroup_value() {
     [ "$status" -eq 0 ]
     check_cgroup_value $CGROUP_BLKIO "blkio.weight" 500
 
+    # update blkio-leaf-weight
+    runc update test_update --blkio-leaf-weight 500
+    check_cgroup_value $CGROUP_BLKIO "blkio.leaf_weight" 500
```

Comment:
Yeah, we can handle this later I guess. We need to do the test framework improvements for the memory cgroup anyway.